src/androidTest/java/com/couchbase/lite/DatabaseEncryptionTest.java

```diff
@@ -230,4 +234,67 @@ public boolean update(UnsavedRevision rev) {
         Assert.assertNotNull(seekrit);
         Assert.assertEquals(1, seekrit.getDocumentCount());
     }
+
+    public void testEncryptedAttachments() throws Exception {
+        if (!isSQLiteDB())
+            return;
+
+        Manager cryptoManager = getEncryptionTestManager();
```

Comment:
`getEncryptionTestManager()` is just return `currentManager`. Why does the method need special naming?